README.md

```diff
@@ -15,6 +15,23 @@ To cite the Manubot project or for more information on its design and history, s
 Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br>
 _Manubot Preprint_ (2019) <https://greenelab.github.io/meta-review/>
 
+## Installation
+
```

Comment:
Let's start off this section with something like the following:

```markdown
If you are using the `manubot` python package as part of a manuscript repository, installation of this package is handled though the Rootstock's [environment specification](https://github.com/manubot/rootstock/blob/master/build/environment.yml).
For other use cases, this package can be installed via pip as follows:
```

Reply:
This language looks good to me.  We can capitalize `Python`.